segment[] voronoi
position[] path
position[] smooth_path
